internal/terminal/terminal_windows_test.go

```diff
@@ -0,0 +1,34 @@
+//go:build windows
+
+package terminal
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsTerminal_Windows(t *testing.T) {
+	// Mock IsTestEnv to bypass the test environment check
+	originalIsTestEnv := IsTestEnv
+	IsTestEnv = func() bool { return false }
+	defer func() { IsTestEnv = originalIsTestEnv }()
```

Comment:
Please consider this


```suggestion
// MockIsTestEnv bypasses the test environment check
func MockIsTestEnv(t *testing. T, value bool) {
	t.Helper ()

	originalIsTestEnv := IsTestEnv
	IsTestEnv = func() bool { return value }
	t.Cleanup(func() { IsTestEnv = originalIsTestEnv })
}


func TestIsTerminal_Windows(t *testing.T) {
	
	MockIsTestEnv(t, false)
```

The mock func could be move to terminal.go